.github/workflows/ci.yml

```diff
@@ -292,12 +292,12 @@ jobs:
 
       - name: Check return code
         run: |
-          if [[ $(cat return_code) == 0 ]]; then exit 1; else exit 0; fi
+          if [[ "$(cat return_code)" == "0" ]]; then exit 1; else exit 0; fi
 
       - name: Check output
         run: |
-          EXPECTED=$(echo -e "--- failure auto_trait_impl_removed: auto trait no longer implemented ---\n--- failure trait_missing: pub trait removed or renamed ---")
-          RESULT=$(cat output | grep failure)
+          EXPECTED="$(echo -e "--- failure auto_trait_impl_removed: auto trait no longer implemented ---\n--- failure trait_missing: pub trait removed or renamed ---")"
```

Comment:
Do the nested quotes work in bash?

Reply:
They do! I know I asked the same question when my own mentor taught me to quote everything like this :)